src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.print("Figure: circle, radius: " + radius);
```

Comment:
Use only eng in messages/code: The draw method should not include language-specific terms like 'Figure: circle'. Instead, you should use a more universal term, such as 'Shape: circle', to keep the code consistent and language-neutral.